site/content/en/docs/administration/advanced/webhooks.md

```diff
@@ -0,0 +1,366 @@
+---
+title: 'Webhooks'
+linkTitle: 'Webhooks'
+description: 'Instructions for working with CVAT Webhooks'
+weight: 80
+---
+
+## Create Webhook
+
+In CVAT you can create webhook for project or for organization.
+For creation, you can use our user interface or direct API calls.
+
+In order to create webhook via an API call, see the swagger documentation. (TO-DO: ADD LINK HERE)
```

Comment:
@kirill-sizov Will you add these links to the PR?

Reply:
Added